packages/contracts/contracts/MarketRegistry.sol

```diff
@@ -755,13 +780,13 @@ contract MarketRegistry is
      * @param _marketId The ID of a market.
      * @return Duration of a loan until it is delinquent.
      */
-    function getPaymentCycleDuration(uint256 _marketId)
+    function getPaymentCycleValue(uint256 _marketId)
         public
         view
         override
         returns (uint32)
     {
-        return markets[_marketId].paymentCycleDuration;
+        return markets[_marketId].paymentCycleValue;
```

Comment:
then lets change this to return both the type and the value instead of adding an additional function for each